### ISSUE TITLE
Lizzie won't load engine

### ISSUE THREAD

**Issue by willow1044** [featurecat/lizzie#699]
I have a new Acer Win 10 laptop with  an AMD Radeon(TM) Vega 8 Graphics card (with openCL 2.0).

When I try and install leela with gpu support the process just dies. No output or error message.
It will however complete with the cpu only version.
However, when I load up Lizzie the UI just says that it is loading the engine and that never completes.

I opened the display window and received the following error:
"Uncaught exception: OpenCL error at C:\Data\Data\Coding\Python\KataGo\cpp\neuralnet\openclhelpers.cpp, func err, line 188, error CL_PLATFORM_NOT_FOUND_KHR"

I checked my system32 folder and there is a openCL.DLL file.

I'd prefer not to return the laptop but I only bought it for this purpose so it's a paperweight otherwise.

**Comment by featurecat** [comment]
I can help troubleshoot this weekend. Are you on Discord? It might be easier to try and resolve the issue in real time and then post the solution here. My username is windcat#1120

**Comment by willow1044** [comment]
Hi, not sure what Discord is so likely not. Is that the discord chat app? I'm keen to get this resolved so if you can give me some more deets then I'm happy to meet up at your convenience. I'm in pacific standard time.

**Comment by boviz** [comment]
What did you figure out as a solving?

**Comment by willow1044** [comment]
I did some googling and found posts online wherre the OP had installed Leela and also listed their GPU. Then I returned the original laptop and went out and purchased a new one with the right type of card. Cost a lot more money but Leela is really awesome so no regrets. 
Use it to review all my games. Find it very helpful.

For completeness sake here are the details:
Product: NVIDIA GeForce GTX 1650
Driver Version: 26.20.100.6911
Operating System: Windows* 10 Home (10.0.18363)
Physical Memory: 8034 MB
Vendor ID: 8086
Device ID: 3E9B
Device Revision: 00
Graphics Output Protocol (GOP) Version: 9.0.1082
Current Resolution: 1920 x 1080

Processor: Intel(R) Core(TM) i5-9300H CPU @ 2.40GHz
Processor Speed: 2400 MHz
Processor Graphics in Use: Intel(R) UHD Graphics 630
Shader Version: 5.1
OpenGL* Version: 4.6
OpenCL* Version: 2.1
Vulkan* Version: 1.1.103
CUI Appx Version: 3.3.0.0

* Microsoft DirectX* *
Runtime Version: 12.0
Hardware-Supported Version: 12.0
